### PR TITLE
Restsharp109

### PR DESCRIPTION
**Pull request by mjkkirschner**
replaces:
https://github.com/DynamoDS/Dynamo/pull/14490

we could not consume 110 in Dynamo when targeting net6.
slack thread for future reference:
https://autodesk.slack.com/archives/C01C8GGT9EY/p1697764048583789

- [x] https://github.com/DynamoDS/PackageManagerClient/pull/89 must be merged first.

### CODE REVIEW

**Review by pinzart90 — APPROVED** [review]

File: src/DynamoCore/DynamoCore.csproj [hunk 1]
```diff
@@ -27,11 +27,14 @@
   </ItemGroup>
   <ItemGroup Label="Common dependencies">
     <PackageReference Include="Autodesk.IDSDK" Version="1.1.6" />
+    <PackageReference Include="DynamoVisualProgramming.LibG_229_0_0" Version="3.0.0.2767" GeneratePathProperty="true" />
+    <PackageReference Include="DynamoVisualProgramming.LibG_230_0_0" Version="3.0.0.2770" GeneratePathProperty="true" />
+    <PackageReference Include="Greg" Version="3.0.0.2926" />
     <PackageReference Include="DynamoVisualProgramming.LibG_229_0_0" Version="3.0.0.2855" GeneratePathProperty="true" />
     <PackageReference Include="DynamoVisualProgramming.LibG_230_0_0" Version="3.0.0.2880" GeneratePathProperty="true" />
     <PackageReference Include="Greg" Version="2.5.0.5076" />
```
Comment (aparajit-pratap) [inline src/DynamoCore/DynamoCore.csproj:35]: Duplicates

File: src/Tools/NodeDocumentationMarkdownGenerator/NodeDocumentationMarkdownGenerator.csproj [hunk 2]
```diff
@@ -12,12 +12,14 @@
   </PropertyGroup>
     <ItemGroup>
         <PackageReference Include="CommandLineParser" Version="2.8.0" />
+        <PackageReference Include="DynamoVisualProgramming.LibG_229_0_0" Version="3.0.0.2767" />
+        <PackageReference Include="Greg" Version="3.0.0.2926" />
         <PackageReference Include="DynamoVisualProgramming.LibG_229_0_0" Version="3.0.0.2855" />
         <PackageReference Include="Greg" Version="2.5.0.5076" />
```
Comment (aparajit-pratap) [inline src/Tools/NodeDocumentationMarkdownGenerator/NodeDocumentationMarkdownGenerator.csproj:18]: ⚠️

**Comment by mjkkirschner** [comment]
@aparajit-pratap thanks for catching these.

File: src/DynamoCore/DynamoCore.csproj [hunk 3]
```diff
@@ -27,11 +27,13 @@
   </ItemGroup>
   <ItemGroup Label="Common dependencies">
     <PackageReference Include="Autodesk.IDSDK" Version="1.1.6" />
+    <PackageReference Include="DynamoVisualProgramming.LibG_229_0_0" Version="3.0.0.2767" GeneratePathProperty="true" />
+    <PackageReference Include="DynamoVisualProgramming.LibG_230_0_0" Version="3.0.0.2770" GeneratePathProperty="true" />
+    <PackageReference Include="Greg" Version="3.0.0.2926" />
     <PackageReference Include="DynamoVisualProgramming.LibG_229_0_0" Version="3.0.0.2855" GeneratePathProperty="true" />
     <PackageReference Include="DynamoVisualProgramming.LibG_230_0_0" Version="3.0.0.2880" GeneratePathProperty="true" />
```
Comment (aparajit-pratap) [inline src/DynamoCore/DynamoCore.csproj:34]: @mjkkirschner I'm still seeing 2 versions of LibG nugets for 229 and 230?

File: src/Tools/NodeDocumentationMarkdownGenerator/NodeDocumentationMarkdownGenerator.csproj [hunk 4]
```diff
@@ -12,12 +12,13 @@
   </PropertyGroup>
     <ItemGroup>
         <PackageReference Include="CommandLineParser" Version="2.8.0" />
+        <PackageReference Include="DynamoVisualProgramming.LibG_229_0_0" Version="3.0.0.2767" />
+        <PackageReference Include="Greg" Version="3.0.0.2926" />
         <PackageReference Include="DynamoVisualProgramming.LibG_229_0_0" Version="3.0.0.2855" />
```
Comment (aparajit-pratap) [inline src/Tools/NodeDocumentationMarkdownGenerator/NodeDocumentationMarkdownGenerator.csproj:17]: Same here, still seeing 2 versions of libg nuget.

Comment (mjkkirschner) [reply src/Tools/NodeDocumentationMarkdownGenerator/NodeDocumentationMarkdownGenerator.csproj:17]: thanks @aparajit-pratap - totally missed that as I did not touch LibG in the nuget UI so did not expect it to be affected, I will fix it.

Comment (aparajit-pratap) [reply src/Tools/NodeDocumentationMarkdownGenerator/NodeDocumentationMarkdownGenerator.csproj:17]: I think you might need to pull the latest master to your PR branch if you haven't already done so before making these commits.

Comment (mjkkirschner) [reply src/Tools/NodeDocumentationMarkdownGenerator/NodeDocumentationMarkdownGenerator.csproj:17]: thanks @aparajit-pratap I need to update to whatever the actual published version of Greg.dll ends up being so I will do that then.

**Review by aparajit-pratap — APPROVED** [review]
LGTM assuming you'll be updating to the final version of Greg soon.

**Comment by mjkkirschner** [comment]
this should now start passing, I will keep an eye on it before merging.

**Pull request merged**